test/src/unit-capi-vfs.cc

```diff
@@ -472,7 +477,11 @@ void VFSFx::check_move(const std::string& path) {
 
 #ifndef _WIN32
 void VFSFx::check_copy(const std::string& path) {
-  // Copy file when running on POSIX
+  // Do not support HDFS
```

Comment:
Ideally we would say something about unsupported S3 since that patch is still a work-in-progress. However, since that will be coming soon, no need to re-run the CI to change a comment.